go-controller/hack/update-codegen.sh

```diff
@@ -56,4 +53,7 @@ done
 
 echo "Generating CRDs"
 mkdir -p _output/crds
-controller-gen crd paths=./pkg/crd/... output:crd:dir=_output/crds
+controller-gen crd:crdVersions="v1"  paths=./pkg/crd/... output:crd:dir=_output/crds
+echo "Editing egressFirewall CRD"
+sed -i -e ':begin;$!N;s/.*metadata:\n.*type: object/          metadata: \n            type: object\n            properties:\n              name:\n                type: string\n                pattern: ^default$/;tbegin;P;D' \
+	_output/crds/k8s.ovn.org_egressfirewalls.yaml
```

Comment:
What is this for? This needs a comment explaining it...

Reply:
done